Client/Frontend/Browser/TabManagerStore.swift

```diff
@@ -95,15 +77,32 @@ class TabManagerStore {
 
         writeOperation.cancel()
 
+        // TODO: Look into this more
         let tabStateData = NSMutableData()
         let archiver = NSKeyedArchiver(forWritingWith: tabStateData)
+//        let simpleTabs = savedTabs.map { BasicSavedTab(tab: $0.toSimpleTab(), isSelected: false) }
```

Comment:
nit: I know this portion is a part of your cleanup but adding a comment as a reminder